lib/tasks/projects.rake

```diff
@@ -194,4 +194,21 @@ namespace :projects do
       end
     end
   end
+
+  desc 'Verify Go Projects'
+  task :verify_go_projects, [:before_date, :limit] => :environment do |_task, args|
+    limit = args.limit || 1000
+    before_date = Date.parse(args.before_date) || Date.today + 1.day
```

Comment:
Wondering if this would need an `after_date` too. I don't think there's a way to tell if we've already checked a project or not.